bucket/duplicati.json

```
@@ -0,0 +1,44 @@
+{
+    "version": "2.0.6.3_beta_2021-06-17",
+    "homepage": "https://www.duplicati.com/",
+    "description": "A free, open source, backup client that securely stores encrypted, incremental, compressed backups on cloud storage services and remote file servers.",
+    "license": "LGPL-2.1",
+    "notes": [
+        "If you want Dupilicati to run at the startup of your system, run:",
+        "& \"$dir\\Duplicati.WindowsService.exe\" install",
+        "",
+        "To remove Duplicati startup, run:",
+        "& \"$dir\\Duplicati.WindowsService.exe\" uninstall"

```

```suggestion
        "If you want Dupilicati to run at the startup of your system, run:",
        "sudo \"$dir\\Duplicati.WindowsService.exe\" install",
        "",
        "To remove Duplicati startup, run:",
        "sudo \"$dir\\Duplicati.WindowsService.exe\" uninstall"
```
I had to run these as admin otherwise they failed with the error: `System.Security.SecurityException: The source was not found, but some or all event logs could not be searched.  Inaccessible logs: Security, State.`

We might need to check for admin before uninstallation then too. Maybe take a look at how the `Everything` manifest deals with the service?